bin/gen_schematic_dot.py

```diff
@@ -1,13 +1,17 @@
-import circuit
```

Comment:
bin/ seems like a good place for this. I would like to keep this.

Reply:
I moved this to bin already. I was asking about the comparator .sp and .dot files. Do they belong to the repo? If so, should I move them under tests and build unit tests around them?